### ISSUE TITLE
[QST]Profiling kernel performance

### ISSUE THREAD

**Issue by wxthu** [NVIDIA/cutlass#422]
Like in Pytorch, I can specify a convolution kernel as **torch.nn.Conv2d**(12,24, 3, padding=..., stride=...), then manually feed some input tensors , run this kernel 10000 times and measure the average latency . Again, I can tune the params and measure latency of new kernel. I wonder that how I can do that in CUTLASS ? 
ps: I failed to find an effective way to meet my needs

**Comment by wxthu** [comment]
BTW, if I want to measure the performance of GEMM, given the customized input tensor, how can I do that?

**Comment by hwu36** [comment]
You can use cutlass profiler.  The best GPU profiler you've ever seen.  https://github.com/NVIDIA/cutlass/blob/master/media/docs/profiler.md

**Comment by wxthu** [comment]
> You can use cutlass profiler. The best GPU profiler you've ever seen. https://github.com/NVIDIA/cutlass/blob/master/media/docs/profiler.md

It looks as if I cannot feed **special  inputs formats** (e.g. block diagonal matrix) into GEMM because I want to test the performance of sparse matrix-sparse matrix multiplication given the **block diagonal pattern**

**Comment by hwu36** [comment]
We dont support block diagonal matrix.
We dont support sparse-sparse gemm
